build_torch_xla_libs.sh

```diff
@@ -26,7 +26,7 @@ if [[ "$XLA_CUDA" == "1" ]] && [[ "$CLOUD_BUILD" == "true" ]]; then
 fi
 
 OPTS=(--cxxopt="-std=c++14")
-if [[ "$CC" =~ ^clang ]]; then
+if [ $(basename -- $CC) == 'clang' ]; then
```

Comment:
This still needs the `=~ ^clang` as compiler names are like `clang++-7` and so forth.

Reply:
Done.